### PR TITLE
fix(intake-formats): properly test event fields

### PR DESCRIPTION
**Pull request by gaelmuller**
Make sure the content of `event.*` fields is correctly compared to expectations.

### CODE REVIEW

**Review by PierrickV — APPROVED** [review]

**Comment by PierrickV** [comment]
image

tests fixed (only some coverage tests are still failing)

File: Imperva/imperva-waf/ingest/parser.yml [hunk 1]
```diff
@@ -12,16 +12,14 @@ pipeline:
...
-        input_field: parse_kv.message.start | float / 1000
+        input_field: parse_kv.message.start
         output_field: date
-        format: timestamp
   - name: parse_end
     external:
       name: date.parse
       properties:
-        input_field: parse_kv.message.end | float / 1000
+        input_field: parse_kv.message.end
```
Comment (squioc) [inline Imperva/imperva-waf/ingest/parser.yml:21]: Nice shoot

File: Google Cloud/Google Cloud Audit/tests/2sv_disable.json [hunk 2]
```diff
@@ -9,16 +9,7 @@
...
...isable\",\n    \"resourceName\": \"organizations/123\",\n    \"metadata\": {\n      \"activityId\": {\n        \"uniqQualifier\": \"-7789616625639281959\",\n        \"timeUsec\": \"1632459962686000\"\n      },\n      \"event\": [\n        {\n          \"status\": {\n            \"success\": true\n          },\n          \"parameter\": [\n            {\n              \"type\": \"TYPE_STRING\",\n              \"label\": \"LABEL_OPTIONAL\",\n              \"value\": \"INfDlrzP9IH8_QE\",\n              \"name\": \"dusi\"\n            }\n          ],\n          \"eventName\": \"2sv_disable\",\n          \"eventType\": \"2sv_change\"\n        }\n      ],\n      \"@type\": \"type.googleapis.com/ccc_hosted_reporting.ActivityProto\"\n    }\n  },\n  \"insertId\": \"-tn3jrd3lko\",\n  \"resource\": {\n    \"type\": \"audited_resource\",\n    \"labels\": {\n      \"service\": \"login.googleapis.com\",\n      \"method\": \"google.login.LoginService.2svDisable\"\n    }\n  },\n  \"timestamp\": \"2021-09-24T05:06:02.686Z\",\n  \"severity\": \"NOTICE\",\n  \"logName\": \"organizations/123/logs/cloudaudit.googleapis.com%2Fdata_access\",\n  \"receiveTimestamp\": \"2021-09-24T05:06:03.845372592Z\"\n}",
```
Comment (squioc) [inline Google Cloud/Google Cloud Audit/tests/2sv_disable.json:12]: Maybe we should have some `event.*` fields in the parser. At least, `event.kind` fro example

File: SkyhighSecurity/skyhigh_secure_web_gateway/ingest/parser.yml [hunk 3]
```diff
@@ -119,7 +119,8 @@ stages:
 
       - set:
           event.type: [ 'connection', 'access', 'allowed' ]
-        filter: '{{set_ecs_fields.event.type == ["allowed"] or parse_kv.message.result == "OBSERVED"}}'
+        filter: '{{ parse_kv.message.block_id == "0" or set_ecs_fields.event.type == ["allowed"] or parse_kv.message.result == "OBSERVED"}}'
```
Comment (squioc) [inline SkyhighSecurity/skyhigh_secure_web_gateway/ingest/parser.yml:122]: No test associated to this fix?

**Review by squioc — DISMISSED** [review]

File: SkyhighSecurity/skyhigh_secure_web_gateway/ingest/parser.yml (hunk 3 above)
Comment (PierrickV) [reply SkyhighSecurity/skyhigh_secure_web_gateway/ingest/parser.yml:122]: https://github.com/SEKOIA-IO/intake-formats/blob/main/SkyhighSecurity/skyhigh_secure_web_gateway/tests/mcafee_access_log_success.json#L19-L23

This is a fix for this part of the test

File: Google Cloud/Google Cloud Audit/tests/2sv_disable.json (hunk 2 above)
Comment (PierrickV) [reply Google Cloud/Google Cloud Audit/tests/2sv_disable.json:12]: I will improve this part in another PR

**Review by squioc — APPROVED** [review]

**Pull request merged**